packages/widget/src/interfaces/index.ts

```diff
@@ -22,6 +22,11 @@ export interface Eip1193Provider {
   }): Promise<unknown>;
 }
 
+export type WalletConnectOptions = {
```

Comment:
don't define your options, use types from web3-onboard [package](https://github.com/blocknative/web3-onboard/blob/develop/packages/walletconnect/src/types.ts#L3)

You could also expose: https://github.com/blocknative/web3-onboard/blob/develop/packages/core/src/types.ts#L33

Reply:
this is weird, I tried to import them but I couldn't do it. I'm going to recheck again

Reply:
You can always extract it using typeof+ArgumentsType